submissions/nkeirukaEnendu/HTML-Forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Page</title>
+</head>
+<body>
+    <header>
+        <h1>NextLine Furnitures</h1>
+    </header>
+   <nav>
+       <div>
+           <ul>
+               <li> <a href="#product">Gallery</a></li>
+               <li> <a href="#">FAQs</a></li>
+               <li> <a href="#contactUs">Contact Us</a></li>
+           </ul>
+       </div>
+   </nav>
+   <div id="product">
+       <img src="./img - Copy/Rectangle 11.png" alt="">
```

Comment:
The image name here Rectangle 11.png is not properly written. Remove the spaces in the image name. Also, your product page task has been merged earlier, you dont have to repeat it in this new Pull request.